src/load_helper.h

```diff
@@ -0,0 +1,58 @@
+#ifndef ADPLUG_LOAD_HELPER_H
+#define ADPLUG_LOAD_HELPER_H
+
+#include <cstring>
+#include <stdint.h>
```

Comment:
Should this be `<cstdint>` seeing as we're in C++ here?

Reply:
no, and that's because `<cstdint>` is a C++11 header. The fact that `<stdint.h>` works is purely coincidence, because C++98 never mentions `<cstdint>` or `<stdint.h>`, and that is due to the presence of C99 headers. Only C++11 uses C99 as a referential standard. The fact that this works is coincidence and not design (and fails on MSVC for instance).

Reply:
This won't work on old Apple compilers, that have C99 headers (`<stdint.h>`) but not C++11 headers (`<cstdint>`) yet.

Reply:
Oh you're right, my mistake.  I have only used `stdint.h` myself and I thought I must have been doing it wrong but that would be why I never saw `cstdint` before!  No argument about using C99 features already, but the fact of the matter is that the code compiles for everyone now, whereas if we make C++11 mandatory I am absolutely sure it will break for *someone*.